### PR TITLE
Replaces Iskoola Pota, Latha, Shonar Bangla fonts in Simorgh to new fonts

### PR DESCRIPTION
**Pull request by pvaliani**
Relates to https://github.com/bbc/simorgh/pull/9434

**Overall change:**
Removes references to the Sinhalese, Tamil and Bengali service fonts which are being replaced

**Code changes:**

- Removes references to test environment
- Removes previous font config info from service files

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/bengali.js [hunk 1]
```diff
@@ -67,7 +60,7 @@ export const service = {
       'https://www.bbc.com/bengali/institutional-50409861#authorexpertise',
     publishingPrinciples: 'https://www.bbc.com/bengali/institutional-50409861',
     isTrustProjectParticipant: true,
-    script,
+    script: bengaliNotoSerif,
```
Comment (MuhammadAHussain) [inline src/app/lib/config/services/bengali.js:63]: Can't you remove the use of script entirely?

Comment (MuhammadAHussain) [reply src/app/lib/config/services/bengali.js:63]: Nevermind, searched up what it does 🤦

**Review by MuhammadAHussain — APPROVED** [review]
LGTM 👍

File: src/app/lib/config/services/bengali.js (hunk 1 above)
Comment (pvaliani) [reply src/app/lib/config/services/bengali.js:63]: Can you expand? The script was originally set for the test environment so we've simply refactored it for live and it is needed in order to pull in the new font from gel foundations

**Review by jroebu14 — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
Checked that Iskoola Pota, Latha , Shonar Bangla font references are removed..

For eg;https://www.test.bbc.com/tamil/access-to-news?morph_env=live

**Comment by paruchurisilpa** [comment]
Checked that Iskoola Pota, Latha , Shonar Bangla font references are removed..

For eg;https://www.test.bbc.com/tamil/access-to-news?morph_env=live

**Pull request merged**